app/src/main/java/org/oppia/android/app/help/thirdparty/LicenseListFragment.kt

```diff
@@ -0,0 +1,42 @@
+package org.oppia.android.app.help.thirdparty
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.oppia.android.app.fragment.InjectableFragment
+import javax.inject.Inject
+
+/** Fragment that contains list of licenses for a third-party in the app. */
+class LicenseListFragment : InjectableFragment() {
+  @Inject
+  lateinit var licenseListFragmentPresenter: LicenseListFragmentPresenter
+
+  companion object {
+    private const val LICENSE_LIST_FRAGMENT_DEPENDENCY_INDEX = "LicenseListFragment.DependencyIndex"
+    fun newInstance(dependencyIndex: Int): LicenseListFragment {
```

Comment:
Kdoc for this as it is a public function.

Reply:
Also one blank line above this

Reply:
Done.